Document/0x07a-Testing-Authentication-and-Session-Management.md

```diff
@@ -316,12 +322,12 @@ All available functions that allow a user to set a password need to be verified,
 - Forgot Password function that allows a user to set a new password or
 - Change Password function that allows a logged in user to set a new password.
 
-An interception proxy should be used, to bypass client passwords checks within the app in order to be able verify the password policy implemented on server side.
+An interception proxy should be used, to bypass client passwords checks within the app in order to be able verify the password policy implemented on server side. More information about testing methods can be found in the OWASP Testing Guide (OTG-AUTHN-007)<sup>[1]</sup>
```

Comment:
Is it really necessary to do password complexity checks on the server? I know client site checks can be bypassed, but this only allows a user to set his own password to something that doesn't match the password policy.